crates/ruff_python_formatter/resources/test/fixtures/black/raw_docstring.py

```diff
@@ -0,0 +1,16 @@
+# flags: --preview --skip-string-normalization
```

Comment:
Should we add an option file that sets `preview: true`?

Reply:
I don't think we support them for black tests yet

Reply:
We do, but only a single options file